src/main/java/ch/njol/skript/classes/data/DefaultComparators.java

```diff
@@ -663,6 +664,10 @@ public Relation compare(EntitySnapshot snap1, EntitySnapshot snap2) {
 		// UUID
 		Comparators.registerComparator(UUID.class, UUID.class, (one, two) -> Relation.get(one.equals(two)));
 		Comparators.registerComparator(UUID.class, String.class, (one, two) -> Relation.get(one.toString().equals(two)));
+
+		// ComponentWrapper
+		Comparators.registerComparator(ComponentWrapper.class, ComponentWrapper.class,
+			((o1, o2) -> Relation.get(o1.getComponent().equals(o2.getComponent()))));
```

Comment:
This worries me. Two items could feasibly be converted to components and determined to be equal despite being completely different items.